applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c

```diff
@@ -43,17 +44,37 @@ static bool delete_broadcast_src;
 
 uint8_t audio_mapping_mask[CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT] = {UINT8_MAX};
 
+/* Advertising data for peer connection */
 #if (CONFIG_AURACAST)
+/* Location of the first byte for the PBA features and parental rating */
+#define PBA_FEATURE_START		(BT_UUID_SIZE_16)
+#define BROADCAST_PARENTAL_RATING_START (BT_UUID_SIZE_16 + 4)
+
 /* Make sure pba_buf is large enough for a 16bit UUID and meta data
  * (any addition to pba_buf requires an increase of this value)
  */
-NET_BUF_SIMPLE_DEFINE(pba_buf, BT_UUID_SIZE_16 + 8);
-static struct bt_data ext_ad[5];
-#else
-static struct bt_data ext_ad[4];
+static uint8_t pba_buf[] = {BT_BYTES_LIST_LE16(0x1856),
+			    0x00,
+			    0x06,
+			    0x02,
+			    BT_AUDIO_METADATA_TYPE_PARENTAL_RATING,
+			    0x00,
+			    0x02,
+			    BT_AUDIO_METADATA_TYPE_AUDIO_STATE,
+			    BT_AUDIO_ACTIVE_STATE_ENABLED};
 #endif /* (CONFIG_AURACAST) */
 
-static struct bt_data per_ad[1];
+/* Broadcast Audio Streaming Endpoint advertising data */
+#define BROADCAST_ID_START (BT_UUID_SIZE_16)
+static uint8_t brdcst_id_buf[BT_UUID_SIZE_16 + BT_AUDIO_BROADCAST_ID_SIZE] = {
+	BT_BYTES_LIST_LE16(BT_UUID_BROADCAST_AUDIO_VAL)};
+/* Buffer for Appearance */
+static uint8_t brdcst_appearance_buf[BT_UUID_SIZE_16];
+/* Buffer for Public Broadcast Announcement */
+NET_BUF_SIMPLE_DEFINE_STATIC(base_buf, 128);
+
+/* Broadcast name */
+static uint8_t brdcst_name_buf[32];
```

Comment:
Maybe add a comment stating that PBP spec requires broadcast name to be 4-32 UTF-8 encoded characters long